jenkins/debian/debos/rootfs.yaml

```diff
@@ -136,6 +138,13 @@ actions:
     chroot: false
     command: mv ${ROOTDIR}/boot/initrd.img-min ${ARTIFACTDIR}/{{ $basename -}}/initrd.cpio.gz
 
+{{ if $extra_packages_remove }}
```

Comment:
OK, I guess that can solve the issue of having unnecessary packages in some rootfs variants.  I'm not entirely sure if this or the negative version of it is the best approach (i.e. listing packages that have to stay rather than ones that can be removed), but either way it seems worth giving a spin now.